### PR TITLE
Add Book -> Book Section and Book Section -> Book to context menu

### LINKED ISSUE #2732: Add "Create Book Section" menu item above "Duplicate Item" for Book items
https://forums.zotero.org/discussion/comment/413762/#Comment_413762

Duplicate, change type to Book Section, add related-items relation, maybe open Title field?

### PR DESCRIPTION
**Pull request by AbeJellinek**
Closes #2732

### CODE REVIEW

File: chrome/locale/en-US/zotero/zotero.properties [hunk 1]
```diff
@@ -359,6 +359,8 @@ pane.items.menu.createParent              = Create Parent Item…
 pane.items.menu.createParent.multiple     = Create Parent Items
 pane.items.menu.renameAttachments = Rename File from Parent Metadata
 pane.items.menu.renameAttachments.multiple = Rename Files from Parent Metadata
+pane.items.menu.duplicateAndConvert.toBookSection = Create Book Section
+pane.items.menu.duplicateAndConvert.toBook = Create Book
```
Comment (dstillman) [inline chrome/locale/en-US/zotero/zotero.properties:363]: I think this one has to be "Create Book Item", even if it's not quite symmetrical. "Create Book Section" on its own seems fine.

Comment (dstillman) [reply chrome/locale/en-US/zotero/zotero.properties:363]: Or "Create Book from Book Section"?

Comment (dstillman) [reply chrome/locale/en-US/zotero/zotero.properties:363]: I think "Create Book Section" and "Create Book from Book Section" would work.

Comment (AbeJellinek) [reply chrome/locale/en-US/zotero/zotero.properties]: Sure, pushed the latter option.

**Comment by dstillman** [comment]
Can we backport this to master?

**Comment by AbeJellinek** [comment]
Sure, I'll open another PR for that.

**Pull request merged**